sandbox-backend/src/main/java/com/exadel/sandbox/team5/mapper/MapperUtil.java

```diff
@@ -0,0 +1,22 @@
+package com.exadel.sandbox.team5.mapper;
+
+import org.modelmapper.ModelMapper;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import java.util.List;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+@Configuration
+public class MapperUtil {
```

Comment:
It is not correct two use this class for two purpose: configuration of beans and utility class. It is better to have separate class with configuration something like that `ApplicationConfiguration` in the separate `config` package.

Reply:
moved to Application